src/components/ComputeModal.js

```diff
@@ -1121,10 +1124,10 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
           ])],
           [willDeletePersistentDisk(), () => h(Fragment, [
             p([
-              'Reducing the size of a persistent disk requires it to be deleted and recreated. This will ',
+              'To reduce the size of the PD, the existing PD will be deleted and a new one will be created and attached to your virtual machine instance. This will ',
               span({ style: { fontWeight: 600 } }, ['delete all files on the disk.'])
             ]),
-            h(SaveFilesHelp)
+            getExistingEnvironmentConfig().runtime.tool === 'RStudio' ? h(SaveFilesHelpRStudio) : h(SaveFilesHelp) 
```

Comment:
The logic makes sense. It seems like there are other places in `ComputeModal` where `SaveFilesHelp` is rendered. We should apply the logic above in those places as well?